aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -0,0 +1,103 @@
+| **2**  | **Body**     | The body may be populated as many questions as appropriate.          | <Checkmark a11yTitle="true" size="small" /> | <span>Question types can include [StarRating](https://v2.grommet.io/starrating), [ThumbsRating](https://v2.grommet.io/thumbsrating), select all that apply (e.g. checkbox group), choose one (e.g. RadioButtonGroup), open ended (e.g. TextArea), or any other input types.</span> |                                                             
+| **3**  | **Footer **  | Contains two actions: "Submit" and "Dismiss" if used in Layer modal. Otherwise only "Submit" is needed inline. | <Checkmark a11yTitle="true" size="small" /> |  Submit button sends survey responses to the HPE customer experience team and initiates thank you message. If presented in a modal, the Dismiss button allows the user to bypass the survey and continue their product experience. |
+
+## Feedback setup and submission
+To iniate user feedback collection, please contact the <a href={`mailto:${contact.email}?subject=Collecting%20user%20feedback%20in%20a%20product%20experience`}>unified customer experience team</a> for the following steps:
+
+1. A dedicated survey will be created for your use case. At that time, an API token, survey id, and response id will be provided.
+1. Obtain the survey response schema with the [retrieve a survey response call](https://api.qualtrics.com/1179a68b7183c-retrieve-a-survey-response). The properties in the "values" object represent the data which should be submitted with each new survey response.
+1. Survey responses are submitted using [create a new response](https://api.qualtrics.com/f1cad92018d2b-create-a-new-response).
+
+```
+const fasdf = () 
+```
+
+## Examples
+
+### Solicited feedback
+Solicited feedback is an active method for asking users for their opinions and perspectives. 
+A feedback questionaire is presented after a user has completed a task or consumed a piece of 
+content. Solicited feedback should not interupt or impede a user from completing their intended 
+objective.
+
+#### Task flow in center-layer
+
+Click on "Power 1 Device" to see the progressive example of the feedback questions. 
+<Example
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/feedback/PowerDeviceFeedback.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/feedback/SolicitedFeedbackExample.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/feedback/PowerOnBodyExample.js',
+  ]}
+  github="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/feedback/SolicitedFeedbackExample.js"
+  showResponsiveControls={['fullScreen']}
+  screenContainer
+  height="650px"
+  width="100%"
+>
+  <SolicitedFeedbackExample />
+</Example>
+
+
+### Capturing unsolicited feedback
```

Comment:
```suggestion
### Unsolicited feedback
```
small bit of simplification to align with "Solicited feedback". soft opinion

Reply:
Im okay with just `Unsolicited feedback` but I believe @halocline added the `Capturing unsolicited` so just want his feedback

Reply:
Soft opinion. Either is fine with me.